public/pages/editor/wizard.html

```diff
@@ -57,6 +57,8 @@
             <label>Number of results</label>
           </div>
           <div class="col-md-6">
+            <select name="operator" ng-model="watcher.$$condition_operator" ng-options="operator.operator as operator.name for operator in watcher.$$condition_operators">
```

Comment:
This select control is different by style from other controls we have. We use bootstrap 3 styles. Please add `class="form-control"` to your select.